### ISSUE TITLE
Typescript removes comments before assignment values.

### ISSUE THREAD

**Issue by Glinkis** [Microsoft/TypeScript#22377]
**Code**
```ts
const value: number = /*1*/ 0 /*2*/;
```

**Expected behavior:**
Should produce
```ts
var value = /*1*/ 0 /*2*/;
```

**Actual behavior:**
Produces
```ts
var value = 0 /*2*/;
```

**Playground Link:**
[link](https://www.typescriptlang.org/play/#src=const%20value%20%3D%20%2F*1*%2F%200%20%2F*2*%2F%3B)

**Comment by j-oliveras** [comment]
Probably solved by #22141. You can try it with a [nightly build](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Nightly%20Builds.md).

**Comment by Glinkis** [comment]
@j-oliveras Yep, seems to be. I'll close this then.
